src/widget/translator.cpp

```diff
@@ -72,6 +73,11 @@ void Translator::translate()
         QCoreApplication::installTranslator(translator);
     }
 
+    // Added to fix #3542
+    QSet<QString> rtlLanguages = {"ar", "he"};
```

Comment:
If it will work, why not. Don't depend on this value being correct though – i.e. check should be for `RTL`, otherwise assume `LTR`.

`\"Translate\" this to 'LTR' in left-to-right` ?
`s/such as/for example/`, or `e.g.` ?